service/src/test/java/bio/terra/workspace/common/BaseUnitTest.java

```diff
@@ -1,12 +1,10 @@
 package bio.terra.workspace.common;
 
-import org.junit.jupiter.api.Tag;
-import org.springframework.test.context.ActiveProfiles;
+import bio.terra.workspace.common.annotations.Unit;
 
 /**
- * Base class for unit tests: not connected to cloud providers. Includes GCP unit tests & others
- * cloud-agnostic WSM component unit tests
+ * Base class for Spring Boot unit tests: not connected to cloud providers. Includes GCP unit tests
+ * & others cloud-agnostic WSM component unit tests
  */
-@Tag("unit")
-@ActiveProfiles("unit-test")
+@Unit
 public class BaseUnitTest extends BaseUnitTestMocks {}
```

Comment:
I'm glad you asked this.  Yes, this is currently reflective of the state of WSM unit tests as a whole: it's a non-trivial lift to convert a test suite that currently uses Spring Boot to one that doesn't.  But ultimately, we do not want the base unit test to include Spring.

To make this more clear, I'll make some additional naming changes up the chain of base tests.

Reply:
Actually, fully propagating these name changes touched 100+ files.  I walked them back and will handle in a follow-on PR if we'd like, to avoid obscuring the meat of this one.